DesignSafe-CI/portal
# task/DES-2029 - Updated documentation link to new location

## Overview: ##
Fixing obsolete link to Recon Portal documentation

## PR Status: ##


## Related Jira tickets: ##

* [DES-2029](https://jira.tacc.utexas.edu/browse/DES-2029)

## Summary of Changes: ##
Changed link to current Recon Portal documentation

## Testing Steps: ##
1. Navigate to https://www.designsafe-ci.org/recon-portal/
2. Click the link right under the words Recon Portal "Learn more about contributing" that should now be directed to https://www.designsafe-ci.org/rw/user-guides/tools-applications/recon-portal/

## UI Photos:

## Notes: ##
